active_plugins/filterobjects_stringmatch.py

```diff
@@ -46,20 +50,37 @@ def create_settings(self):
         self.filter_out = Alphanumeric(
             "What string to filter out",
             "AAAA",
-            doc="""Enter a name for the measurement calculated by this module.""",
+            doc="""Enter the string that should be used to filter objects.""",
         )
 
+        self.filter_method = Choice(
+            "Filter method",
+            [METHOD_EXACT, METHOD_CONTAINS],
+            doc="""Select whether to only filter objects that are an exact match for the string entered
+            (e.g. Object 'AAAAB' will NOT be filtered by string 'AAAA') 
+             or to filter any object that contains the string entered
+             (e.g. Object 'AAAAB' will be filtered by string 'AAAA').""",
+        )
+
+        self.filter_column = Measurement("Measurement",
+                lambda: "Image",
```

Comment:
This should be the object name, see https://github.com/CellProfiler/CellProfiler/blob/f0fda05ed77e76781c75c83aeb404ad031780b10/src/frontend/cellprofiler/modules/filterobjects.py#L443-L452